lib/errors.js

```diff
@@ -7,6 +7,14 @@ class NotYetImplementedError extends ES6Error {
   }
 }
 
+class NotImplementedError extends ES6Error {
+  constructor () {
+    super("That method is not implemented");
+    this.jsonwpCode = 15; // TODO use status package
```

Comment:
this should be the same as UnknownError since we are just making this up and it's not part of the spec